packages/js/scripts/copy-typedefs.js

```diff
@@ -13,7 +13,7 @@ function copyFileSync( source, target ) {
     }
   }
 
-  fs.writeFileSync(targetFile, fs.readFileSync(source, 'utf8').replace(/'\.\//g, '\'./types/'));
+  fs.writeFileSync(targetFile, fs.readFileSync(source, 'utf8'));
```

Comment:
I don't know why this was here. I had to remove it in order to have correct type definitions.